README.md

```diff
@@ -67,9 +67,9 @@ Move the iterator to the previous break position before the given code unit inde
 
 Return the code unit index of the most recently discovered break position, as an offset from the beginning of the string. Initially the `index` is 0.
 
-#### `get %SegmentIterator%.prototype.breakType`
+#### `get %SegmentIterator%.prototype.precedingSegmentType`
 
-The `breakType` of the most recently discovered segment. If there is no current segment (e.g., a just-instantiated SegmentIterator, or one which has reached the end), or if the break type is "grapheme", then this will be `undefined`.
+The `precedingSegmentType` of the segment which precedes the current iterator location in logical order. If there is no preceding segment (e.g., a just-instantiated SegmentIterator), or if the break type is "grapheme", then this will be `undefined`.
```

Comment:
```suggestion
The type of the segment which precedes the current iterator location in logical order. If there is no preceding segment (e.g., a just-instantiated SegmentIterator), or if the granularity is "grapheme", then this will be `undefined`.
```